platform/openide.util/src/org/openide/util/MapFormat.java

```diff
@@ -152,7 +152,7 @@ protected Object processKey(String key) {
     * parser found unmatched brackets (this exception should be switched off
     * using setExactMatch(false)).
     */
-    public String processPattern(String newPattern) throws IllegalArgumentException {
+    private String processPattern(String newPattern) throws IllegalArgumentException {
```

Comment:
This is a public method on an exposed API, and sigtests warns about about an incompatibility with previous versions of the API:

```java
email: api-changes@netbeans.org
SignatureTest report
Base version: 9.23
Tested version: 9.25
Check mode: bin [throws removed]
Constant checking: on


Class org.openide.util.MapFormat
  "E1.2 - API type removed" : method public java.lang.String org.openide.util.MapFormat.processPattern(java.lang.String)

BUILD FAILED
/home/travis/build/apache/netbeans/nbbuild/build.xml:1885: The following error occurred while executing this line:
/home/travis/build/apache/netbeans/nbbuild/templates/projectized.xml:778: Signature tests return code is wrong (1), check the messages above
```